layouts/partials/openapi/render-object-table.html

```diff
@@ -97,6 +107,27 @@
 
     {{ end }}
 
+ {{/* If additionalProperties is true, do nothing.
+      If additionalProperties is nil, treat is as the default (true) and do nothing.
+      If additionalProperties is false, raise a build-time error:
+      "Everything is extensible in Matrix unless otherwise noted".
+      Otherwise, it should be an OpenAPI schema object. Include the description
+      of that schema object, if it exists. */}}
+ {{ if (and (ne $additionalProperties nil) (ne $additionalProperties true)) }}
+  {{ if (and (reflect.IsMap $additionalProperties)) -}}
+   {{ if (index $additionalProperties "description") -}}
+  <tr class="row-additional-properties">
+   <td colspan="3">
+    <em>Notes on additional properties:</em> {{ $additionalProperties.description }}
+   </td>
+  </tr>
+   {{ end }}
+  {{/* TODO: should we handle the case where additional properties are
+             permitted, but must follow an explicit schema? */}}
+  {{ else if (not $additionalProperties) -}}
+   {{ errorf "Unexpected additionalProperties=false for %s" $title }}
```

Comment:
it's unclear to me that we should be raising an error in this case. Isn't it legitimate to specify that a given object may *not* have additional properties, even if that is currently unused?

Reply:
:shrug: There's some context here: https://github.com/matrix-org/matrix-spec/pull/1162#discussion_r914153682 ; I don't have any strong opinions here.